generators/spring-boot/templates/src/main/java/_package_/_entityPackage_/_partials_entity_/get_all_template.ejs

```diff
@@ -78,7 +78,11 @@ _%>
     public <% if (reactive) { %>Mono<<% } %>List<<%= instanceType %>><% if (reactive) { %>><% } %> getAll<%= entityClassPlural %>(<% if (fieldsContainNoOwnerOneToOne) { %>@RequestParam(name = "filter", required = false) String filter<% } %><% if (implementsEagerLoadApis && fieldsContainNoOwnerOneToOne) { %>,<% } %><% if (implementsEagerLoadApis) { %>@RequestParam(name = "eagerload", required = false, defaultValue = "true") boolean eagerload<% } %>) {<%- include('get_all_stream_template', {viaService: viaService}); -%>
         log.debug("REST request to get all <%= entityClassPlural %>");
     <%_ if (viaService) { _%>
+      <% if (implementsEagerLoadApis && !dtoMapstruct) { %>
+        return <%= entityInstance %>Repository.findAllWithEagerRelationships()<% if (reactive) { %>.collectList()<% } %>;
```

Comment:
`findAllWithEagerRelationships` should be used for collections where we should avoid multiple bags.
This error looks related to the repository since the relationships are not collections.

Reply:
What's the best way to proceed with getting this merged? Is it good enough for now?

Reply:
Looks like `findAllWithEagerRelationships` is used bellow, but this is ignoring the service layer and executing the repository.
It's probably required to add a parameter at service layer to make it optional like it's in the rest layer.